src/xrc/src/lib.rs

```diff
@@ -63,8 +63,8 @@ pub const XRC_IMMEDIATE_REFUND_CYCLES: u64 = 5_000_000_000;
 /// The base cost in cycles that will always be charged when using the `xrc` canister.
```

Comment:
This comment is not really correct unless "using" means "getting a valid response".

Reply:
Updated the comment.